orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/LogoutService.java

```diff
@@ -80,21 +82,15 @@ public LogoutService(
 
     private APIGatewayProxyResponseEvent generateLogoutResponse(
             URI logoutUri,
-            Optional<String> state,
             TxmaAuditUser auditUser,
             Optional<String> clientId,
             Optional<String> rpPairwiseId) {
         LOG.info("Generating logout response using URI: {}", logoutUri);
```

Comment:
Not sure we should be logging the state here as it's arbitrary RP controlled input and we aren't sure it's not PII

Reply:
Sure, only logging the path now.